### PR TITLE
auto-detect a user's preferred language

### PR DESCRIPTION
**Pull request by yoshi-taka**
For more information, see the following link: 
https://blog.logrocket.com/advanced-localization-techniques-vue-js

### CODE REVIEW

File: web/src/i18n/i18n.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function loadLocaleMessages() {
 }
 
 export default new VueI18n({
-  locale: process.env.VUE_APP_I18N_LOCALE || 'en',
+  locale: navigator.language.split('-')[0] || 'en',
```
Comment (ann-kilzer) [inline web/src/i18n/i18n.js:20]: From the link you shared, perhaps this is more complete:

```suggestion
  locale: navigator.language.split('-')[0] || process.env.VUE_APP_I18N_LOCALE || 'en',
```

**Review by ann-kilzer — COMMENTED** [review]
Thanks for the suggestion! Currently we only support English and Japanese. What happens when the browser locale is a language unsupported by our site?

We may wish to add this alongside the upcoming work in #88.

**Comment by yoshi-taka** [comment]
Thanks you for the comment.
You have set the fallback_locale. It's the language that is used if the requested translation is not found.

https://kazupon.github.io/vue-i18n/guide/fallback.html

**Comment by ann-kilzer** [comment]
Tested by setting my browser preferences to Deutsch. Looks great. Thanks for the contribution @yoshi-taka

**Pull request merged**